.github/workflows/java.yml

```diff
@@ -0,0 +1,62 @@
+name: Gradle CI/CD
+
+on:
+  push:
+    branches: [ actions ]
+  pull_request:
+    branches: [ actions ]
+
+jobs:
+  build_and_test:
+  
+    runs-on: ubuntu-latest
+
+    services:
+      postgres:
+        image: postgres
+        env:
+          POSTGRES_PASSWORD: Qwe123
```

Comment:
Proszę używać w takim celu secretów. Nie należy podawać haseł otwartym tekstem 

https://github.com/Azure/actions-workflow-samples/blob/master/assets/create-secrets-for-GitHub-workflows.md

Reply:
Nie mam dostępu do ustawień tego repozytorium